packages/share_plus/share_plus/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +8,7 @@
         android:icon="@mipmap/ic_launcher"
         android:label="share_example">
         <activity
-            android:name="io.flutter.embedding.android.FlutterActivity"
+            android:name="io.flutter.embedding.android.FlutterFragmentActivity"
```

Comment:
Is there any reason why the example should be changed?

Reply:
In order to test that the `requestCode` is in the correct range.